pkg/rhcos/updatepayload/updatepayload.go

```diff
@@ -0,0 +1,157 @@
+// Package updatepayload extracts the RHCOS build version from an
```

Comment:
nit: clayton has been trying to scrub using term `update payload` and and consistently use `release image`

Reply:
why is this part of `pkg/rhcos` and not `pkg/`

Reply:
> nit: clayton has been trying to scrub using term `update payload`...

Fixed in e189de133 -> 3a99f85b8.

> why is this part of `pkg/rhcos` and not `pkg/`

containers/image is a heavy dependency, and I want folks to be able to use the existing pkg/rhcos without needing to import that code.  On the other hand, this is certainly RHCOS-related.  So a subpackage of the existing pkg/rhcos made the most sense to me, but I can move it if you like (just tell me the package path to use).